components/command/package.go

```diff
@@ -47,8 +47,13 @@ func (m *GenericPackageManager) Ensure(packageRef string, transform Transformer,
 
 		opts = append(opts, utils.PulumiDependsOn(updateDB))
 	}
+	var cmdStr string
+	if checkCmd != "" {
+		cmdStr = fmt.Sprintf("bash -c '%s || %s %s'", checkCmd, m.installCmd, packageRef)
```

Comment:
Commands are already run in a shell. With the current package managers we have, it should be enough.

Reply:
I like `command` to check if the binary already exists, I will go for it.
However without changing how we format the command: [here](https://github.com/DataDog/test-infra-definitions/blob/main/components/command/unixOSCommand.go#L99) we need to wrap our command in `bash-c` otherwise the formatted command is:
`sudo command -v docker || apt-get install docker.io`
Which does not work because sudo applies only to `command -v`
I could modify this [line](https://github.com/DataDog/test-infra-definitions/blob/main/components/command/unixOSCommand.go#L99) to wrap everyhing in `bash -c` tho

Reply:
Ah right the `sudo` wrapping. LGTM